README.markdown

```diff
@@ -298,6 +298,9 @@ Build the source with this module:
          --add-module=/path/to/ngx_devel_kit \
          --add-module=/path/to/lua-nginx-module
 
+# Note that you also should add arguments which are used in your current nginx  version.
```

Comment:
"also should" => "should also"
2 spaces between `nginx` and `version`, which should be one.
"arguments" => "options"